src/AsmResolver.DotNet/Collections/Parameter.cs

```diff
@@ -76,6 +76,11 @@ public TypeSignature ParameterType
         /// <inheritdoc />
         public string Name => Definition?.Name ?? GetDummyArgumentName(MethodSignatureIndex);
 
+        /// <summary>
+        /// Creates a <see cref="ParameterDefinition"/> for this parameter and adds it to the method definition if it does not exist.
+        /// </summary>
+        public void CreateParameterDefinition() => _parentCollection?.CreateParameterDefinition(this);
```

Comment:
Should we rename it to `CreateDefinition` for brevity and let it return the created `ParameterDefinition`? It is already clear from the context that this definition is going to be a parameter definition, and it would allow for immediately using the created definition as a variable as well.